src/app.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const fs = require('fs/promises');
+const path = require('path');
+const [sourceFile, destinationFile] = process.argv.slice(2);
+
+function copy(src, dest) {
+  const srcPath = path.join(__dirname, `/${src}`);
+  const destPath = path.join(__dirname, `/${dest}`);
```

Comment:
No need to join `__dirname` here. Because of this, you can't copy files outside the `src` directory. E.g. running this from project root doesn't work: `node src/app.js  readme.md readme.md`.